thanos/plugindefinition.yaml

```
@@ -64,9 +64,13 @@ spec:
           name: thanos.query.serviceLabels
           required: false
           type: "map"
-        - default: 
-          - prometheus-operated:10901
-          description: Standard port to be exposed by Prometheus Operator. If multiple Prometheis are deployed, this needs to be switched to prometheus' CRD name suffixed with -prometheus.
+        - default: false
+          description: Removes standard Thanos endpoints for overarching Queries 
+          name: thanos.query.standalone
+          required: false
+          type: "bool"
```

NIT: The `" "` around the type are not needed although it's a string. Applies to all fields.
```suggestion
          type: bool
```